lib/integrations/jira.rb

```diff
@@ -0,0 +1,84 @@
+require "integrations/base"
+
+module Integrations
+  class Jira < Base
+    def self.key
+      'jira'
+    end
+
+    def send_event
+      return false unless has_failed_tests?
```

Comment:
run_error and run_webhook_timeout don't have a failed test in the payload. Does it support all the events?